### ISSUE TITLE
Request for error handling example

### ISSUE THREAD

**Issue by caschbre** [icflorescu/trpc-sveltekit-example#15]
I was reading through the [tRPC Error Handling docs](https://trpc.io/docs/error-handling), but wasn't quite sure how to implement that using trpc-sveltekit.

I figured this example project would be a great place for that. :-)

**Comment by icflorescu** [comment]
What do you mean? Like [here](https://github.com/icflorescu/trpc-sveltekit-example/blob/main/src/routes/authors/%2Bpage.svelte#L69)?

**Comment by caschbre** [comment]
Ah, sorry I wasn't very clear. I was looking at the error handling example provided in the trpc docs. 

```javascript
export default trpcNext.createNextApiHandler({
  // ...
  onError({ error, type, path, input, ctx, req }) {
    console.error('Error:', error);
    if (error.code === 'INTERNAL_SERVER_ERROR') {
      // send to bug reporting
    }
  },
});
```

**Comment by icflorescu** [comment]
Ah, I see, you mean handling on the server. I did not expose `onError` as a parameter of `createTRPCHandle`, but I guess I should.
Will release a new version shortly.

**Comment by icflorescu** [comment]
Landed in `v3.1.0`.
Look at the bottom of the page here: https://icflorescu.github.io/trpc-sveltekit/handling-errors
Basically you can now provide an `onError` function to `createTRPCHandle`.

**Comment by caschbre** [comment]
Awesome!  Exactly what I was looking for. Thanks!
